packages/next/next-server/server/render.tsx

```diff
@@ -404,6 +404,14 @@ export async function renderToHTML(
             `Warning: A page's revalidate option was set to more than a year. This may have been done in error.\nTo only run getStaticProps at build-time and not revalidate at runtime, you can set \`revalidate\` to \`false\`!`
           )
         }
+      } else if (data.revalidate === false) {
+        // `false` is an allowed behavior. We'll catch `revalidate: true` and
+        // fall into our default behavior.
+      } else {
+        // By default, we revalidate after 1 second. This value is optimal for
+        // the most up-to-date page possible, but without a 1-to-1
+        // request-refresh ratio.
+        data.revalidate = 1
```

Comment:
I'm still not sure if this is the best idea

Reply:
I thought this was the value we settled on after discussion with @rauchg. One second seems to be the best value if we don't want the user to have to configure it -- I think it should "just work" out-of-the-box (not require you specify it), and this value does that.

Reply:
It's also the biggest "wow factor" 🤔. As in, "wow my website updates so fast but it's still a CDN HIT", 😆.